src/FileWriteHandler.cpp

```diff
@@ -0,0 +1,42 @@
+#include "FileWriteHandler.hpp"
+
+/**
+ * @brief Construct a new FileWriteHandler object
+ *
+ * @param fileSystemPolicy File system policy object. Can be mocked if needed.
+ */
+FileWriteHandler::FileWriteHandler(const FileSystemPolicy& fileSystemPolicy)
+	: m_fileSystemPolicy(fileSystemPolicy)
+{
+}
+
+std::string FileWriteHandler::execute(const std::string& path, const std::string& content)
```

Comment:
Comment could be added to describe the functionality of ``execute``

Reply:
Good finding, added